package.json

```
@@ -13,7 +13,8 @@
   "scripts": {
     "build": "tsc",
     "clean": "rimraf lib",
-    "prepublish": "npm run build"
+    "prepublish": "npm run build",
+    "watch": "tsc -w --jsx react"
```

This config should be part of `tsconfig.json` instead if it isn't already there

---

The `--jsx react` option can be omitted since its already in `tsconfig.json`, but I still need a `watch` target here for `jlpm watch` to work, right?

```suggestion
    "watch": "tsc -w"
```